.github/compressed-size-action.yml

```diff
@@ -0,0 +1,14 @@
+name: Compressed Size
+
+on: [ pull_request ]
```

Comment:
maybe, we need to create a new pull request?

Reply:
Yeah, only pulls would show this data, 
though don't you think a file on the repo works better since we know the state of every commit with updated `size.json` cause PR's will get hard to search after a point

Reply:
It has pros and cons, but so far this has been more pain with multiple contributors. I wanted to switch to github actions.

Reply:
Make sense, we can kinda have github actions create a size file and push to the repo as well, I have something similar working for https://github.com/barelyhuman/custom-rom-index , so if we want to look into that angle, let me know